### ISSUE TITLE
Homepage create html structure

### ISSUE THREAD

**Issue by marcusmoller97** [EmpyreanMist/Kino---Cinema#42]
# Create a structure that are easily accessible and easy to read for the user.
 
* The structure should be semantically correct to be able to be used with voiceover.
* It should be used with header, nav, section, article and footer. Avoid using div other than for design purposes later.
    * For inspiration see **https://www.figma.com/design/SeXL3dMcv83md8jcKd3ElW/Kino?t=0pZ8O2KIurnvniQf-0**

**Comment by marcusmoller97** [comment]
# Create a structure that are easily accessible and easy to read.
 
* The structure should be semantically correct to be able to be used with voiceover.
* It should be used with header, nav, section, article and footer. Avoid using div other than for design purposes later.
